src/cdk-experimental/ui-patterns/tabs/BUILD.bazel

```diff
@@ -0,0 +1,19 @@
+load("//tools:defaults2.bzl", "ts_project")
+
+package(default_visibility = ["//visibility:public"])
+
+ts_project(
```

Comment:
We should avoid using the `glob` when possible - just list out the source files

Reply:
I'll be sure to go back and clean that up for the rest of the ui patterns

Reply:
Yeah, we do `glob` everywhere but in my experience it just makes the BUILD harder to fix when there's an issue. I brought it up because I had to redo the `src/material/core` BUILD because of the globs